docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -99,91 +159,116 @@ for /l %x in (1, 1, 1000) do clc -c source.yaml map --name my-map set key-%x val
-----
-<1> If you don't have a Hazelcast Enterprise license key, you can either use a Hazelcast Viridian https://docs.hazelcast.com/cloud/free-trial[trial cluster] or request a xref:getting-started:enterprise-overview.adoc#step-2-get-your-enterprise-license[trial license].
-<2> If you want to use the full distribution, remove the `-slim` postfix. See xref:getting-started:editions.adoc#full-and-slim-distributions[Distributions] to learn about the differences between full and slim distributions.
+. Navigate to the folder in which you extracted the DMT package
+. Open the _migration_config/source/hazelcast.yaml_ file in your favorite editor
+. Update the `cluster-name` field to match the name of your source cluster
+. Update the `cluster-members` field to match the addresses of the cluster members
+. Save the file
 
-The cluster tries to start on port `5901` if it's available. It chooses another port otherwise. You can check the port in the cluster logs shown on the terminal.
+To update the data structure information, complete the following steps:
 
-The configuration file for the target cluster is `target.yaml`, which can be found in the folder where you extracted the DMT package.
-In case the cluster starts on a different port as you notice within the cluster logs, you must change the `address` field in `target.yaml` accordingly to match the port number.
+. Navigate to the folder in which you extracted the DMT package
+. Open the _migration_config/data/imaps.yaml_ and/or the _migration_config/data/imaps.yaml_ file in your favorite editor
+. Update the file content to match the names of your maps
++
+NOTE: If you have multiple data structures, use a new line for each map name.
 
-NOTE: Instead of Docker, you can also by https://hazelcast.com/get-started/download/[download] Hazelcast 5.3.2 or newer, request a xref:getting-started:enterprise-overview.adoc#step-2-get-your-enterprise-license[trial license], and xref:getting-started:get-started-binary.adoc#step-2-start-a-local-member[start] a cluster.
+. Save the file
 
-=== Step 4. Start the Migration Cluster
+=== Check the Target Cluster
 
-Run the following command in the folder where you extracted the DMT package.
+Ensure that the target cluster is running on one of the following:
 
-[source,shell]
-----
-$ HZ_NETWORK_PORT_PORT=5702 HZ_CLUSTERNAME=migration ./bin/hz start
-----
+* Enterprise Edition version 5.3.2 or later 
+* {hazelcast-cloud}
+
+=== Start the Migration Cluster
+
+To start the migration cluster, complete the following steps:
+
+. Open a terminal
+. Navigate to the folder in which you extracted the DMT package
+. Enter the following command:
++
+====
+HZ_NETWORK_PORT_PORT=5702 HZ_CLUSTERNAME=migration ./bin/hz start
+====
+
+If the specified port is available, the cluster starts on that port. Otherwise, the cluster selects a different port. You can confirm the port used by the cluster in the logs displayed in your terminal.
```

Comment:
> Otherwise, the cluster selects a different port

Maybe point them to [this doc section](https://docs.hazelcast.com/hazelcast/5.3/clusters/network-configuration#port) for more info on port selection. Otherwise it sounds like some random port is selected

Reply:
Reworded and added link